src/Stratis.Bitcoin.IntegrationTests/BlockStore/BlockStoreTests.cs

```diff
@@ -26,121 +27,69 @@ public BlockStoreTests()
-                {
-                    var lst = new List<Block>();
-                    for (int i = 0; i < 30; i++)
-                    {
-                        // roughly 1mb blocks
-                        var block = new Block();
-                        for (int j = 0; j < 3000; j++)
-                        {
-                            var trx = new Transaction();
-                            block.AddTransaction(new Transaction());
-                            trx.AddInput(new TxIn(Script.Empty));
-                            trx.AddOutput(Money.COIN + j + i, new Script(Guid.NewGuid().ToByteArray()
-                                .Concat(Guid.NewGuid().ToByteArray())
-                                .Concat(Guid.NewGuid().ToByteArray())
-                                .Concat(Guid.NewGuid().ToByteArray())
-                                .Concat(Guid.NewGuid().ToByteArray())
-                                .Concat(Guid.NewGuid().ToByteArray())));
-                            trx.AddInput(new TxIn(Script.Empty));
-                            trx.AddOutput(Money.COIN + j + i + 1, new Script(Guid.NewGuid().ToByteArray()
-                                .Concat(Guid.NewGuid().ToByteArray())
-                                .Concat(Guid.NewGuid().ToByteArray())
-                                .Concat(Guid.NewGuid().ToByteArray())
-                                .Concat(Guid.NewGuid().ToByteArray())
-                                .Concat(Guid.NewGuid().ToByteArray())));
-                            block.AddTransaction(trx);
-                        }
-                        block.UpdateMerkleRoot();
-                        block.Header.HashPrevBlock = lst.Any() ? lst.Last().GetHash() : Network.Main.GenesisHash;
-                        lst.Add(block);
-                    }
-
-                    Stopwatch stopwatch = new Stopwatch();
-                    stopwatch.Start();
-                    blockRepo.PutAsync(lst.Last().GetHash(), lst).GetAwaiter().GetResult();
-                    var first = stopwatch.ElapsedMilliseconds;
-                    blockRepo.PutAsync(lst.Last().GetHash(), lst).GetAwaiter().GetResult();
-                    var second = stopwatch.ElapsedMilliseconds;
-                }
-            }
-        }
-
         [Fact]
         public void BlockRepositoryPutBatch()
         {
-            using (var dir = TestDirectory.Create())
+            using (var blockRepo = new BlockRepository(Network.Main, TestBase.CreateDataFolder(this), DateTimeProvider.Default, this.loggerFactory))
             {
-                using (var blockRepo = new BlockRepository(Network.Main, dir.FolderName, DateTimeProvider.Default, this.loggerFactory))
+                blockRepo.SetTxIndexAsync(true).Wait();
+
+                var lst = new List<Block>();
+                for (int i = 0; i < 5; i++)
                 {
-                    blockRepo.SetTxIndexAsync(true).Wait();
+                    // put
```

Comment:
is this in the right place ?  doesn't seem to tally with the line or the block of code